app/Http/Requests/EInvoice/Peppol/StoreEntityRequest.php

```diff
@@ -81,8 +81,8 @@ public function rules(): array
             'acts_as_receiver' => ['required', 'bool'],
             'acts_as_sender' => ['required', 'bool'],
             'tenant_id' => ['required'],
-            'classification' => ['required', 'string'],
-            'vat_number' => [Rule::requiredIf(fn() => $this->input('classification') !== 'individual')],
+            'classification' => ['required', 'in:business,individual'],
+            'vat_number' => [Rule::requiredIf(fn() => $this->input('classification') === 'business')],
```

Comment:
There are more classifications than just business / individual, the rule is

!= individual  classification must be business

Reply:
Got it :+1: